src/app/containers/Metadata/index.jsx

```diff
@@ -19,6 +19,31 @@ const allTags = tags => {
   return aboutTags.concat(mentionTags);
 };
 
+const getTitle = promo =>
+  promo.subType === 'IDX' ? promo.name : promo.headlines.seoHeadline;
```

Comment:
Can you use deepGet here please, feels a bit brital, to I appreciate it wasnt much better before :P